src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedXml.cs

```diff
@@ -232,6 +241,24 @@ private byte[] GetCipherValue(CipherData cipherData)
             throw new CryptographicException(SR.Cryptography_Xml_MissingCipherData);
         }
 
+        private void DownloadCipherValue(CipherData cipherData, out Stream inputStream, out Stream decInputStream, out WebResponse response)
+        {
+            // maybe a network stream, make sure we allow just what is needed!!
+            PermissionSet ps = SecurityManager.GetStandardSandbox(_evidence);
+            ps.PermitOnly();
```

Comment:
that will throw in .NET Core: https://github.com/dotnet/corefx/blob/986ffdb1053d334b862f96b53c67ca3bae222219/src/System.Security.Permissions/src/System/Security/PermissionSet.cs#L41